openslides_backend/models/meeting.py

```diff
@@ -16,10 +18,254 @@ class Meeting(Model):
     collection = Collection("meeting")
     verbose_name = "meeting"
 
+    def get_settings_keys(self) -> List[str]:
+        return [
+            "name",
```

Comment:
Missing welcome title and welcome text.

Reply:
Left it out intentionally, since they don't belong to the traditional settings since they are not on the settings page. They can be added in a future PR. However, I do concur (if nobody says something against it) that they can be handled in the same action.

Reply:
Maye lets avoid creating an extra action just for these two fields.

Reply:
Maybe avoid adding an extra action just for these two fields.